.github/workflows/ts-checks.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
       with:
         submodules: 'false'
     - uses: './.github/actions/pnpm-install'
-    - run: 'pnpm run check'
+    - run: 'pnpm run check:ci'
```

Comment:
- [ ] Resolve this comment once #777 has merged and has been cherry picked into here